package.json

```diff
@@ -3,25 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
     "classnames": "^2.3.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
-    "react-router-dom": "^6.10.0",
+    "react-router-dom": "^6.11.2",
     "react-scripts": "5.0.1",
     "redux": "^4.2.1",
-    "sass": "^1.60.0",
-    "web-vitals": "^2.1.4"
+    "sass": "^1.62.1",
+    "uuid": "^9.0.0",
+    "web-vitals": "^3.3.1"
```

Comment:
why do we need this dependency?

Reply:
It can measure user experience metrics.
removed. as I'm not using it.
Thank you.